app/model/editions/templates/TemplateHelpers.scala

```diff
@@ -39,3 +40,58 @@ object TemplateHelpers {
 
 }
 
+case object EditionType extends Enumeration() {
+
+  type EditionType = Value
+  val Regional, Special, Training = Value
+
+  implicit val readsEditionType: Reads[templates.EditionType.Value] = Reads.enumNameReads(EditionType)
+  implicit val writesEditionType: Writes[templates.EditionType.Value] = Writes.enumNameWrites
+}
+
+case class Header (title: String, subTitle: Option[String] = None)
+object Header {
+  implicit val readsHeader: Reads[Header] = Json.reads[Header]
+  implicit val writesHeader: Writes[Header] = Json.writes[Header]
+}
+
+trait EditionDefinition {
+  val title: String
+  val subTitle: String
+  val edition: String
+  val header: Header
+  val editionType: EditionType
+}
+trait EditionDefinitionWithTemplate extends EditionDefinition {
+  val template: EditionTemplate
+}
+object EditionDefinition {
+  def apply(
+    title: String,
+    subTitle: String,
+    edition: String,
+    header: Header,
+    editionType: EditionType
+  ): EditionDefinition = EditionDefinitionRecord(title, subTitle, edition, header, editionType)
+
+  def unapply(x: EditionDefinition): Option[(String, String, String, Header, EditionType)]
+    = Some(x.title, x.subTitle, x.edition, x.header, x.editionType)
+
+  implicit val formatEditionDefinition: OFormat[EditionDefinition] = Json.format[EditionDefinition]
+  implicit val writesEditionDefinition: OWrites[EditionDefinition] = Json.writes[EditionDefinition]
+  implicit val readsEditionDefinition: Reads[EditionDefinition] = Json.reads[EditionDefinition]
```

Comment:
Are reads and writes needed if we specify a format?

Reply:
Ah... apparently not.  Done.